src/module_methods/book.rb

```diff
@@ -0,0 +1,72 @@
+require_relative '../books'
+require_relative '../label'
+require_relative '../genre'
+require_relative './label'
+require_relative './music_genre'
+require_relative '../storage_helper_methods/storage'
+
+module BookModule
+  include Storage
+  include LabelModule
+  include MusicGenreModule
+
+  def list_books(books)
+    if books.empty?
+      puts "\n Your Books list is empty"
+    else
+      puts "\n Books list - count(#{books.count}):"
+      puts "-----------------------------------------\n"
+
+      books.each_with_index do |book, index|
+        puts "#{index + 1}) Title: '#{book.label.title}'",
+             #  "   Author: #{book.author.first_name}, #{book.author.last_name} ",
```

Comment:
- [ ] Good job so far 👏 👏 Your codes looks great. I really liked it 💯 However, in order to keep it neat and easy to read **I would suggest that you remove all the unnecessary commented-out codes like this one above.** This is among the best practices that every developer should follow 😎